packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -144,7 +144,9 @@ const node4 = mesh.addVirtualNode('node4', {
       },
     },
   },
-  accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout'),
+  accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout',
```

Comment:
could we still have a `accessLog` that does not pass in a `LoggingFormat`?

Reply:
Added that test

Reply:
I personally don't want to increase number of integ tests for a single field which may cause longer test time